### ISSUE TITLE
Update to webpack 5

### ISSUE THREAD

**Issue by isaacabraham** [SAFE-Stack/SAFE-template#472]

**Comment by theimowski** [comment]
I recall there were same issues for Feliz and Webpack 5 - https://github.com/pmmmwh/react-refresh-webpack-plugin/issues/330 
Not sure if it relates to stuff we have in template though @Zaid-Ajaj

**Comment by mattgallagher92** [comment]
Summary from a discussion between me, @theimowski and @MangelMaxime:

* react-refresh requires that code follows certain conventions, such as not exposing types or functions that are not React components
* Feliz code that generates JavaScript code that doesn't follow those conventions will not be able to benefit from react-refresh
* In that case, the behaviour will be just the same as it was for webpack 4 (tab reload)

In summary, there is no problem moving to webpack 5. [fulma-demo](https://github.com/MangelMaxime/fulma-demo) is already using webpack 5, so we can use that as a template.

**Comment by theimowski** [comment]
Closed by #485
